### DISCUSSION TITLE
Twitch Chat & Streamer notifications

### DISCUSSION THREAD

**Discussion by RadianceRS** [runelite/runelite#15050] [Ideas]
There's already been made a twitch chat plugin, which overlaps the runescape chat itself.

There's aswell already been made a plugin for streamers, where you can setup an account to pm you when someone donates, follows or subscribes to your channel (This is Gudi's meme plugin)

Is there any way for two of these plugins to be combined into one, and added to the Runelite main client?

**Comment by Nightfirecat** [comment]
I'm only aware of the client's Twitch plugin, I don't know what Gudi's plugin is or how it works. Any such plugin should probably be added to the plugin hub if it were to be added.
